Commands/StartCommand.php

```diff
@@ -32,13 +32,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $config = $this->initializeConfig($input, $output);
 
-        $class = isset($config['processmanager']) ? $config['processmanager'] : ProcessManager::class;
+        $class = $config['processmanager'] ?? ProcessManager::class;
```

Comment:
The null coalescing operator is only avaiable since php 7.0 or higher. ``composer.json`` allows php 5.6 or higher. This results in a BC break.

Reply:
7.0 is anyway required by this PR.